common/stash/src/main/java/com/bazaarvoice/emodb/common/stash/StashSplitIterator.java

```diff
@@ -46,7 +49,7 @@
     protected Map<String, Object> computeNext() {
         String line;
         try {
-            line = _in.readLine();
+            line = _reader.readLine();
```

Comment:
Looked at the [guava source](https://github.com/google/guava/blob/0ba7ccf36f5384a321cb78d62375bf7574e7bc24/guava/src/com/google/common/io/LineReader.java#L71) to verify.  `LineReader#readLine()` reads in chunks to a local buffer so it is potentially vulnerable to running out of heap but not the `BufferedReader#readLine()` vulnerability.